### PR TITLE
Anqi-UI

### PR DESCRIPTION
**Pull request by Anqi-nus**
Added UI.
Changed performance to sync with team structure.

### CODE REVIEW

File: src/main/java/seedu/command/PerformanceCommandInterpreter.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        case "view_student_result":
+            new ViewStudentResult(commandParameters).execute();
+            break;
+        case "view_assignment_result":
+            new ViewAssignmentResult(commandParameters).execute();
+            break;
+        default:
+            throw new DukeException("UNKNOWN COMMAND");
+        }
+    }
```
Comment (lowjiayee) [inline src/main/java/seedu/command/PerformanceCommandInterpreter.java]: Shouldn't this function return a command object to Duke, then Duke will execute() the command?

**Review by lowjiayee — COMMENTED** [review]
Commands should be returned and executed in Duke, not in Interpreter.

**Review by lowjiayee — COMMENTED** [review]
Looks good

**Comment by benchan911** [comment]
Ready to push

**Pull request merged**